ha/core/controllers/pcs/node_controller.py

```diff
@@ -99,7 +103,33 @@ def status(self, nodeids: list = None) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":{}}}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        all_nodes_status = dict()
```

Comment:
move node status logic to PcsController class

and call it with super and reformat if need

Reply:
added function nodes_status in pcs-controller to avoid confusion between status within node-controller. using that function call in start function of pcsVmNodeController.